build.gradle

```diff
@@ -242,7 +242,7 @@ subprojects {
   }
 
   // Remove the relevant project name once it's converted to JUnit 5
-  def shouldUseJUnit5 = !(["api", "connect", "core", "file", "generator", "runtime", "examples",
+  def shouldUseJUnit5 = !(["api", "connect", "core", "generator", "runtime", "examples",
```

Comment:
You also have to update the JUnit dependency in the build, check one of the recent related commits for an example.